coordinator/src/validator.rs

```diff
@@ -213,7 +213,11 @@ impl Decodable for TransactionWithGas {
 }
 
 pub enum Evidence {
-    DoubleVote, // Should import and use DoubleVote type defined in tendermint module?
+    DoubleVote {
```

Comment:
I wanted not to pass the signature to the application.
Since evidence verification is done by the host, the application does not need information to verify the evidence. 
It only needs information to penalize validators.
This is what Tendermint ABCI does.

Do you think this behavior is problematic? Or are you not satisfied with the name Evidence?

Reply:
If you are not satisfied with the name, I can rename it to something like`VerifiedCrime`

Reply:
Since the name `Evidence` is defined a long time ago, I'll just create a new commit to rename it.